### PR TITLE
Change helm version to 2.16.10 from 3

### PR DESCRIPTION
**Pull request by viveksinghggits**
## Change Overview

This PR changes helm version from build docs image to 2.16.10 from 3.

## Pull request type

Please check the type of change your PR introduces:

- [x] :hamster: Trivial/Minor
- [x] :bug: Bugfix

## Issues

- #XXX

## Test Plan

NA

### CODE REVIEW

**Review by ankitjain235 — APPROVED** [review]

File: docker/docs-build/Dockerfile [hunk 1]
```diff
@@ -13,4 +13,4 @@ RUN curl "https://s3.amazonaws.com/aws-cli/awscli-bundle-${AWS_VERSION}.zip" -o
     ./awscli-bundle/install -i /usr/local/aws -b /usr/local/bin/aws
 
 # add helm
-COPY --from=alpine/helm:3.1.0 /usr/bin/helm /usr/local/bin/
+COPY --from=alpine/helm:2.16.10 /usr/bin/helm /usr/local/bin/
```
Comment (pavannd1) [inline docker/docs-build/Dockerfile:16]: What is the reason for this change?

Comment (viveksinghggits) [reply docker/docs-build/Dockerfile:16]: we were running the respective command from shippable node Pavan, where we have helm v2. but now we are running that command from this image. That is the reason we want to change that to v2.
